app/v4/listPeriodSummaries/ListPeriodSummariesConnector.scala

```diff
@@ -45,7 +45,7 @@ class ListPeriodSummariesConnector @Inject() (val http: HttpClient, val appConfi
 
     val downstreamUri =
       if (taxYear.useTaxYearSpecificApi) {
-        TaxYearSpecificIfsUri[DownstreamResp](
+        IfsUri[DownstreamResp](
           s"income-tax/${taxYear.asTysDownstream}/$nino/self-employments/$businessId/periodic-summaries"
         )
       } else if (featureSwitches.isDesIf_MigrationEnabled) {
```

Comment:
Remove this feature switch `featureSwitches.isDesIf_MigrationEnabled` with `featureSwitches: SeBusinessFeatureSwitches` thanks. It is already enabled in [production](https://github.com/hmrc/app-config-production/blob/main/self-employment-business-api.yaml#L67) so should be removed as part of these changes. Also, remove `isDesIf_MigrationEnabled` entirely, thanks.